examples/01-basic-static-list/expected-output.md

```
@@ -0,0 +1,164 @@
+NAME          TYPE     ITEMS   AGE
+basic-list    static   3       5s
+
+$ kubectl get configmap basic-list-data
+NAME              DATA   AGE
+basic-list-data   3      5s
+```
+
+## 2. ConfigMap Contents
+
+```bash
+$ kubectl get configmap basic-list-data -o yaml
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)